systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1029,8 +1030,76 @@ void checkContainerConfiguration(String podNamePrefix, String containerName, Str
             pods.forEach(pod -> {
                 pod.getSpec().getContainers().stream().filter(c -> c.getName().equals(containerName))
                         .forEach(container -> {
+                            List list = container.getEnv().stream().map((Function<EnvVar, Object>) EnvVar::getName).collect(Collectors.toList());
+                            assertTrue(list.contains(configKey));
+
                             container.getEnv().stream().filter(envVar -> envVar.getName().equals(configKey))
-                                    .forEach(envVar -> assertEquals(config, envVar.getValue()));
+                                    .forEach(envVar -> {
+                                        LOGGER.info("Check specific configuration: {}", envVar.getName());
+                                        for (Map.Entry<String, Object> entry : config.entrySet()) {
+                                            assertThat(envVar.getValue().contains(entry.getKey() + "=" + entry.getValue() + "\n"), is(true));
+                                        }
+                                    });
+                        });
+            });
+        } else {
+            fail("Pod with prefix " + podNamePrefix + " in name, not found");
+        }
+    }
+
+    /**
+     * Verifies container configuration by environment key
+     * @param podNamePrefix Name of pod where container is located
+     * @param containerName The container where verifying is expected
+     * @param config Expected configuration
```

Comment:
Again, can wwe be more specific what exqctly is the expected config? Is that a map with environment variables here? Or something else?